datasets/br/br_slavery/br_slavery.yml

```
@@ -0,0 +1,21 @@
+title: Brazil Ministry of Labour and Employment slavery prevention list
+entry_point: crawler.py
+prefix: br-slavery
```

Bit of a nitpick, but you could probably shorten this like so

```suggestion
prefix: br-slv
```